spec/features/data_access/single_granule_access_spec.rb

```diff
@@ -38,10 +38,27 @@
 
     it 'limits the data access to only the selected granule' do
       click_link 'Expand List'
+      wait_for_xhr
       expect(page).to have_content 'FIFE_RAIN_30M.72981621.r30'
     end
   end
 
+  context 'while the user is viewing a single granule of a multi-granule collection' do
```

Comment:
Now you're on the right track. This does do everything I was looking for, but you put it in the wrong place. The context right above this one (line 22) already loads the page and clicks the single granule link. You can nest contexts. The nested context's `before` block will take over the state of the page where the parent context left it, so you don't have to repeat all the setup in code, and it executes the tests faster because you don't have to do a full page load from the `load_page` call.